src/common/helpers/vpc-helper.ts

```diff
@@ -211,6 +219,37 @@ export function suppressEncryptedLogWarnings(flowLog: FlowLog) {
   ]);
 }
 
+export function buildSecurityGroup(
```

Comment:
Is this change related to this PR of removing deprecated construct? Looks like it's adding a new feature.

Reply:
Yes it is related to this PR. This function was located in a different helper file:

```import { buildSecurityGroup } from './kendra-helper';```

So moving it into the vpc helper file